src/include/reservation.h

```diff
@@ -124,7 +118,7 @@ struct resv_alter {
 	char *ra_select;
 	long ra_state;
 	unsigned long ra_flags;
-	struct resv_alter_revert ra_revert;
+	char *rr_select; /* used to revert select*/
```

Comment:
Change this to ra_select_revert.  Add 'for future occurrences' to the end of the comment.

Reply:
not sure of 'rr' goes with '_revert'. How about select_revert?

Reply:
The prefix of all members of a structure should be the same.  It's an indication to the user what structure it is at its use.  That's why it needs to start with 'ra_'